timed/employment/models.py

```diff
@@ -39,6 +39,28 @@ class Employment(models.Model):
     start_date       = models.DateField()
     end_date         = models.DateField(blank=True, null=True)
 
+    @classmethod
+    def employment_at(cls, user, date):
+        """Get the employment on a date for a user.
+
+        :returns: The employment on the date for the user
+        :rtype:   timed.employment.models.Employment
+        """
+        try:
+            return cls.objects.get(
+                (
+                    models.Q(end_date__gte=date) |
+                    models.Q(end_date__isnull=True)
+                ),
+                start_date__lte=date,
+                user=user
+            )
+        except Exception:
```

Comment:
In general generic exceptions should not be caught or thrown...
better I guess here would be to catch Employment.DoesNotExist and raise ParseError